spec/rubocop/config_loader_spec.rb

```diff
@@ -599,8 +605,8 @@ def enabled?(cop)
           # Enabled in grandparent config, department disabled in parent.
           expect(enabled?('Metrics/PerceivedComplexity')).to be(false)
 
-          # Pending in default config, department disabled in grandparent.
-          expect(enabled?('Lint/StructNewOverride')).to be(false)
+          # Pending in grandparent config, department disabled in parent.
+          expect(enabled?('Naming/FileName')).to be(false)
```

Comment:
The previous test was for a cop that wasn't actually `pending` anymore, so I updated it to not rely on the default config not changing.